pkg/apis/monitoring/v1/types.go

```diff
@@ -889,7 +889,7 @@ type ServiceMonitor struct {
 // +k8s:openapi-gen=true
 type ServiceMonitorSpec struct {
 	// Chooses the label of the Kubernetes `Endpoints`.
-	// Its value will be used for the `job`-label's value of the created metrics.
+	// Its value which comes from the same name service's label will be used for the `job`-label's value of the created metrics.
 	//
 	// Default & fallback value: the name of the respective Kubernetes `Endpoint`.
```

Comment:
I believe that the default value is the name of the associated service?

Reply:
> I believe that the default value is the name of the associated service?

Yeah, but actually this value as same as endpoint's name

Reply:
sure but the prometheus configuration defaults to the service's name so I think it's better to mention "service" rather than "endpoint".

https://github.com/prometheus-operator/prometheus-operator/blob/86df45bbbf699f23271007797297792389fef24f/pkg/prometheus/promcfg.go#L1309-L1313